utils/utils_test.go

```diff
@@ -21,6 +23,8 @@ func TestValidateInputKey(t *testing.T) {
 		"45678901jbf",
 		"asasa/123!12/312313",
 		"1231/1231*2/312313",
+		"user/username/phone",
```

Comment:
I'm not sure about it but ok.

Reply:
I forced this pattern here https://github.com/AirHelp/treasury/pull/106/files#diff-cd575165f84eed10d10eb1460b0927b4R15-R16
so this is not a valid string